### DISCUSSION TITLE
is it a risk for my main computer ?

### DISCUSSION THREAD

**Discussion by xgithx** [telekom-security/tpotce#1685] [Q&A]
Because the vps where I have installed the t-pot doesn't have any graphic cards (GPU), I must connect to the honeypot through RDP. I connect to the t-pot honeypot directly from my main computer. Are there any risks for my main computer, because of this? Could the honeypot attacker, attack my main computer through the rdp client, or find my public ip, from the main computer?

**Comment by n0zk1** [comment]
No, there aren't any risks of connecting via RDP to your VPS - You shouldn't also not be worried about getting attacked on you main computer through the RDP client or have your Main computer, Public IP leaked.
